os/adding-users.md

```diff
@@ -44,6 +67,6 @@ To assign an SSH key, run:
 update-ssh-keys -u user1 user1.pem
 ```
 
-## Further Reading
+## Further reading
 
 Read the [full cloud-config]({{site.baseurl}}/docs/cluster-management/setup/cloudinit-cloud-config/) guide to install users and more.
```

Comment:
This isn't part of my patch. I was going to hit these liquid templates in another pass.